pkg/component/autoscaling/hvpa/crd.go

```diff
@@ -7,23 +7,31 @@ package hvpa
 import (
 	"context"
 	_ "embed"
+	"time"
 
+	v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
```suggestion
	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Maybe you can also add something like https://github.com/gardener/gardener/blob/ddde6b6121857c3a68cbcb88a4638d59930c79e2/.golangci.yaml.in#L69-L70 to enable linting